pkgs/tools/misc/peruse/default.nix

```diff
@@ -57,5 +57,6 @@ stdenv.mkDerivation (finalAttrs: {
     license = licenses.gpl2Only;
     maintainers = with maintainers; [ peterhoeg ];
     mainProgram = "peruse";
+    platforms = platforms.linux;
```

Comment:
Thanks for this. How about this instead:

```
inherit (kirigami2.meta) platforms;
```

It presumably works wherever kirigami does.

Reply:
Sounds good.